subsys/net/ip/net_if.c

```diff
@@ -1986,9 +1986,9 @@ u32_t net_if_ipv6_calc_reachable_time(struct net_if_ipv6 *ipv6)
 #define join_mcast_nodes(...)
```

Comment:
Typo in commit msg "s/IPv6/IPv4"
Also better imho "There is no need to implement a dummy function for each as if IPv4 is not enabled then these functions are unused in the core."

Reply:
Now there is duplicate sentences in commit msg. I suggest we remove the "There is no need to implement a dummy function for each: if IPv4 is not enabled, these functions are unused in the core." as that was what I suggested in my comment. Same for the IPv6 patch.

Reply:
indeed, c/p issue